frontend/src/metabase-lib/limit.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  current_limit as _current_limit,
+  limit as _limit,
+} from "cljs/metabase.lib.core";
```

Comment:
This isn't bad here, but this feels like it's going to get really clunky in larger files. Is there a reason (style conventions?) not to do `import * as ML` or similar?

Reply:
Does it still? I thought that was an early issue because the `lib.limit/*` functions weren't being re-exported in `src/metabase/lib/core.cljc` (they are now).

Reply:
@bshepherdson just tried it once more after rebasing on the latest `master`, and I'm still getting a `ML.limit is not a function` kind of error when I'm doing `import * as ML from "metabase.lib.core"